lms/db/__init__.py

```diff
@@ -1,13 +1,12 @@
 import logging
-from functools import cached_property
 
 import alembic.command
 import alembic.config
 import sqlalchemy
 import zope.sqlalchemy
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.inspection import inspect
-from sqlalchemy.orm import Session, sessionmaker
+from sqlalchemy.orm import sessionmaker
```

Comment:
Or this?

Reply:
I can't spot the problem with a short look around .pylintrc